sphinx_automodapi/tests/example_module/public.py

```diff
@@ -0,0 +1,5 @@
+from ._private import Camelot
```

Comment:
Can we have a test where we import from a non-private module? I mean those cases work, so I just want to make sure that keeps being the case.
(although I see cases where they listed in multiple `__all__`s, probably those should be cleaned up, and (which is beyond scope here) preferably errored out by sphinx-automodapi?)

Reply:
Sounds good, will adjust the test